### PR TITLE
Update contributors files and contributors workflow

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* Updating the contributor's workflow to combine project page

### List of changes proposed in this PR (pull-request)

* Add projects from #2017 
* Update the test files for the weekly GitHub action

### What should a reviewer concentrate their feedback on?

- [ ] Confirm that the projects will like to be listed
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: collaborating_projects.md [hunk 1]
```diff
@@ -69,3 +71,71 @@ Details of each members with their contributions have been listed alphabetically
 
 * Personal highlights:
 > I have personally contributed to _The Turing Way_ by drafting chapters in the guide for Reproducible Research, reviewed other contributor's Pull Requests and mentored contributions from Netherlands eScience Center.
+
+## [FAIR Cookbook](https://fairplus.github.io/the-fair-cookbook/content/home.html)
+
+FAIR Cookbook is an online resource that helps researchers and data managers professionals make their data Findable, Accessible, Interoperable and Reusable (FAIR).
+FAIRPlus Cookbook builds on _The Turing Way_ project and community models, and provides chapters as "recipes" according to the FAIR elements, audience type , reading time, and level of difficulty.
```
Comment (sgibson91) [inline collaborating_projects.md:78]:
```suggestion
FAIRPlus Cookbook builds on _The Turing Way_ project and community models, and provides chapters as "recipes" according to the FAIR elements, audience type, reading time, and level of difficulty.
```

File: tests/contributors.sh [hunk 2]
```diff
@@ -7,23 +7,22 @@ contributors_highlight="./contributors.md"
...
+echo '(contributors-record-highlights)=' > $contributors_record
+cat $contributor_highlights >> $contributors_record
 
 # # Get linenumber where all contributors list starts
-echo '\n(contributors-record-all)=' >> $contributor_record
-echo '\n✨Using [all-contributors](https://allcontributors.org/) specification, _The Turing Way_ Recognises all contributors, not just the ones who push code. ✨' >> $contributor_record
-echo '\n# All Contributors\n' >> $contributor_record
+echo '\n(contributors-record-all)=' >> $contributors_record
+echo '\n# All Contributors\n' >> $contributors_record
+echo '\n✨Using [all-contributors](https://allcontributors.org/) specification, _The Turing Way_ Recognises all contributors, not just the ones who push code. ✨' >> $contributors_record
```
Comment (sgibson91) [inline tests/contributors.sh:17]:
```suggestion
echo '\n✨Using [all-contributors](https://allcontributors.org/) specification, _The Turing Way_ recognises all contributors, not just the ones who push code. ✨' >> $contributors_record
```

**Review by sgibson91 — APPROVED** [review]
❤️

**Comment by malvikasharan** [comment]
You're a star, Sarah. While I have you here, should we add Binder and Jupyter Book to the collaborating org page? I have asked Callum to provide Binder details #2017.

**Comment by sgibson91** [comment]
Sorry I didn't get back to you in time! 🌺

**Comment by malvikasharan** [comment]
It was not urgent! I want to share the published version with you, Callum and some other projects to see how best we can highlight the projects that work in sync/collab with The Turing Way. <3

**Pull request merged**